### PR TITLE
Redirect for Users who visit ProfileBuilder but aren't logged in.

### PR DESCRIPTION
**Pull request by samuelsandoval1**
## Proposed Changes
Created redirect condition for users who aren't logged in users that try to go to the profileBuilder page by url.

## Types of Changes
What types of changes does your code introduce to Techish Project?
_Put an x in the boxes that apply_

- [x]  Bugfix (non-breaking change which fixes an issue)
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  Documentation Update (if none of the other choices apply)

## Checklist
Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  I have added necessary documentation (if appropriate)
- [ ]  Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/webapp/ts/profileBuilder.ts [hunk 1]
```diff
@@ -15,4 +22,22 @@ function setForm(value: string) {
             userTypeButton.setAttribute('value','Mentor');
         }
     }
-} 
\ No newline at end of file
+} 
+
+async function redirectifLoggedOut() {
```
Comment (hadleyshapland) [inline src/main/webapp/ts/profileBuilder.ts:27]: nit: function name should be camel case: redirectIfLoggedOut

File: src/main/webapp/ts/profileBuilder.ts [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+    logoutUrl: string;
+    hasProfile: boolean;
+};
+
 function setForm(value: string) {
     const menteeForm =  document.getElementById('Mentee');
     const mentorForm = document.getElementById('Mentor');
     const userTypeButton = document.getElementById('user-input');
 
+
```
Comment (hadleyshapland) [inline src/main/webapp/ts/profileBuilder.ts:12]: nit: delete extra blank line

File: src/main/webapp/ts/profileBuilder.ts (hunk 1 above)
Comment (samuelsandoval1) [reply src/main/webapp/ts/profileBuilder.ts]: Renamed the function to fulfill the camel case

File: src/main/webapp/ts/profileBuilder.ts (hunk 2 above)
Comment (samuelsandoval1) [reply src/main/webapp/ts/profileBuilder.ts]: Deleted

**Pull request merged**